node/src/effect.rs

```diff
@@ -815,17 +815,21 @@ impl<REv> EffectBuilder<REv> {
     pub(crate) async fn request_proto_block(
         self,
         block_context: BlockContext,
+        next_block_height: u64,
         random_bit: bool,
     ) -> (ProtoBlock, BlockContext)
     where
         REv: From<BlockProposerRequest>,
     {
         let deploys = self
             .make_request(
-                |responder| BlockProposerRequest::ListForInclusion {
-                    current_instant: block_context.timestamp(),
-                    past_blocks: Default::default(), // TODO
-                    responder,
+                |responder| {
+                    BlockProposerRequest::ListForInclusion(ListForInclusionRequest {
+                        current_instant: block_context.timestamp(),
+                        past_deploys: Default::default(), // TODO
```

Comment:
Shouldn't we already pass those deploys in? B/c it's still a bug in the code. If not in this PR, do we have a ticket for it?

Reply:
I think that would be [NDRS-294](https://casperlabs.atlassian.net/browse/NDRS-294).

Reply:
Tagged this ticket for the next sprint, so at least we don't forget about it.

Reply:
Gathering these deploys might be somewhat hard. I don't know if we have a ticket, but I'd prefer to ticket it rather than expand the scope of this PR.